src/Util.cpp

```diff
@@ -37,6 +37,31 @@ std::string getImageId(json isd) {
   return id;
 }
 
+
+std::vector<double> getGeoTransform(json isd) {
+  std::vector<double> transform = {};
+  try {
+    transform = isd.at("geotransform").get<std::vector<double>>();
+  } catch (std::exception &e) {
+    std::string originalError = e.what();
+    std::string msg = "Could not parse the geo_transform. ERROR: " + originalError + isd.dump();
+    throw std::runtime_error(msg);
+  }
+  return transform;
+}
+
+
+std::string getProjectionString(json isd) {
```

Comment:
This also needs to be renamed to `getProjection` to match the h file

Reply:
isd how the test didn't have an undefined symbol error????